### PR TITLE
Do not show recurrence summary when repeat is set to NONE

### PR DESCRIPTION
**Pull request by georgehrke**
Related to #1576

### CODE REVIEW

File: src/components/Editor/Repeat/RepeatSummary.vue [hunk 1]
```diff
@@ -92,6 +92,15 @@ export default {
...
+		 * We do not want to show it if it doesn't repeat
+		 *
+		 * @returns {boolean}
+		 */
+		display() {
+			return this.frequency !== 'NONE'
+		},
 		summary() {
 			if (this.frequency === 'NONE') {
 				return this.$t('calendar', 'Does not repeat')
```
Comment (tcitworld) [inline src/components/Editor/Repeat/RepeatSummary.vue]: So…this isn't shown anymore then? Might as well remove it.

Comment (georgehrke) [reply src/components/Editor/Repeat/RepeatSummary.vue]: Indeed. Let me remove that

Comment (georgehrke) [reply src/components/Editor/Repeat/RepeatSummary.vue]: Actually no, we need this.

The RepeatSummaryReadOnly component extends RepeatSummary: https://github.com/nextcloud/calendar/blob/master/src/components/Editor/Repeat/RepeatSummaryReadOnly.vue#L37

And this is also shown when the repeat is non-recurring.

Comment (tcitworld) [reply src/components/Editor/Repeat/RepeatSummary.vue]: Please add a comment so this isn't forgotten, then. :+1:

**Pull request merged**